src/elements/kw-share-card/kw-share-card.html

```diff
@@ -13,131 +12,183 @@
 <dom-module id="kw-share-card">
     <template>
         <style>
-            :host {
-                height: 250px;
-                width: 250px;
-                color: #6e6e6e;
-                border: 1px solid #e0e0e0;
-                border-radius: 3px;
+             :host {
+                display: block;
                 @apply --layout-vertical;
                 @apply --layout-center;
                 @apply --layout-justified;
                 overflow: hidden;
                 @apply --kw-share-card;
-                background: white;
-                box-sizing: border-box;
+            }
+            kw-share-shell{
+                width: 100%;
             }
             :host(:focus) {
-                outline: none;
-                box-shadow: 0 3px 0 0 rgba(0,0,0,0.08);
+				outline: none;
+				box-shadow: 0 3px 0 0 rgba(0, 0, 0, 0.08);
+			}
+            .bottom {
+                @apply --layout-vertical;
+                min-height: 30px;
+                margin-bottom: 16px;
+                margin: 16px;
             }
+            
             iron-image.cover {
                 @apply --layout-flex;
                 z-index: 0;
                 width: 100%;
+                height: 480px;
                 background: inherit;
+                border-radius: 3px;
             }
-            .top, .bottom {
+            
+            .stats {
                 @apply --layout-horizontal;
-                align-self: stretch;
-                min-height: 30px;
-                boz-sizing: border-box;
+                color: #435055;
+                font-size: 12px;
+                margin-bottom: 12px;
             }
-            .top {
-                padding: 16px 16px 7px 16px;
-            }
-            .bottom {
-                padding: 7px 16px 16px 16px;
-                margin-bottom: 0px;
-                @apply --layout-vertical;
-            }
-            .bottom hr {
-                margin: 0;
-                border: 0px;
-                border-bottom: 1px solid lightgrey;
-            }
-            .share-info {
-                @apply --layout-flex;
-                @apply --layout-horizontal;
-                @apply --layout-end;
-                width: 100%;
+            
+            .stats .likes,
+            .stats .comments,
+            .stats .shares {
+                height: 20px;
             }
-            .datetime {
-                @apply --layout-horizontal;
-                @apply --layout-end;
+            .stats .likes span,
+            .stats .comments span,
+            .stats .shares span{
+                padding-right: 16px;
             }
-            .title {
-                color: black;
-                font-weight: bold;
-                margin-right: 12px;
+            .stats .likes span img,
```

Comment:
Hi Paul, they have it on share card too
![image](https://cloud.githubusercontent.com/assets/3015181/20677143/05862086-b58a-11e6-8ff9-c2bac24c8d44.png)